packages/fast-figma-plugin-msft/spec.md

```diff
@@ -0,0 +1,29 @@
+# Problem statement
+Different teams use different design tools to design user interfaces. In an ideal world, we could write one plugin that would work across all design tools, however APIs and capabilities across these tools are different. Our goal is to define a generic controller model where implementation details are abstracted away from the primary controller logic. This will allow us to use the same controller logic in any design tool, leaving only a small code footprint that needs to be written for each plugin ecosystem. This will also allow us to more easily test the system, because impelmentation details touching individual design-tool APIs will be abstracted.
+
+# How

```

Comment:
in theory when markdown gets converted, # represents the heading level and since we cant have two h1s on a page I'd suggest making this `##`

Reply:
heh this can go away, this was just for organizing thoughts